clap_builder/src/parser/features/suggestions.rs

```diff
@@ -13,13 +13,17 @@ where
     T: AsRef<str>,
     I: IntoIterator<Item = T>,
 {
+    use rapidfuzz::distance::jaro;
+    let scorer = jaro::BatchComparator::new(v.chars());
+    // Confidence of 0.7 so that bar -> baz is suggested
+    let args = jaro::Args::default().score_cutoff(0.7);
```

Comment:
```suggestion
    use rapidfuzz::distance::jaro_winkler;
    let scorer = jaro_winkler::BatchComparator::new(v.chars());
    // Confidence of 0.7 so that bar -> baz is suggested
    let args = jaro_winkler::Args::default().score_cutoff(0.8);
```
This could be reverted to the original behavior using Jaro-Winkler as well